src/helpers/signed-out-fallback.js

```diff
@@ -5,11 +5,27 @@ import { withCurrentUser } from 'src/components/withCurrentUser';
 import AuthViewHandler from 'src/views/authViewHandler';
 
 const Switch = props => {
-  const { Component, FallbackComponent, ...rest } = props;
+  const { Component, FallbackComponent, searchParamException, ...rest } = props;
+  let isException = false;
+
+  if (searchParamException) {
+    const searchParamsString = (props.location && props.location.search) || '';
+    const searchParams = new Map(
+      searchParamsString
+        .substring(1)
+        .split('&')
+        .map(kv => kv.split('='))
+    );
```

Comment:
No need for using a Map here, let's use the `query-string` module we already use across our codebase:

```JS
import queryString from 'query-string';

const searchParams = queryString.parse(searchParamsString);
const isException = searchParams[searchParamsException] && searchParams[searchParamsException] !== false;
```

Reply:
@mxstbr @rafaelalmeidatk i want to add suggestion. The login page still will be functional. we can disable by simply covering whole login page with `overlay` with transparent background. But i am not sure how that fit in this solution

Reply:
Honestly for this first version it is probably fine if the login page keeps working 👍

Reply:
Done, but for some weird reason, I can't log in anymore in my local project so I couldn't test it.